lib/src/runner/configuration/load.dart

```diff
@@ -537,7 +564,12 @@ class _ConfigurationLoader {
     _validate(node, "$name must be a map.", (value) => value is Map);
     var loader = new _ConfigurationLoader(node, _source,
         global: _global, runnerConfig: runnerConfig ?? _runnerConfig);
-    return loader.load();
+    var config = loader.load();
+    var include = (node.value as YamlMap)['include'];
+    if (include != null) {
+      config = _processIncludeIfAny(config, node);
+    }
```

Comment:
Could this logic be moved into `_ConfigurationLoader.load()`, so it doesn't need to be duplicated between here and the top-level `load()` method?

Also, if `_processIncludeIfAny()` is in `_ConfigurationLoader`, it can use the `_parseNode()` helper method which encapsulates some of the logic it's doing.

Reply:
I'm not 100% sure of the best way to get it to use _parseNode. Do you want to take a look at the current state and suggest changes? I did move and refactor into `.load`. PTAL.